_source/_posts/2024-08-15-otp-over-sms.md

```diff
@@ -30,7 +30,7 @@ The one-time passcode (OTP) you send using SMS or Voice may not go to the phone
 
 * Longer login times than other methods
 
-Okta [recommended moving away](/blog/2020/05/why-you-should-ditch-sms-as-an-auth-factor/) from [SMS/Voice authentication](/blog/2020/05/why-you-should-ditch-sms-as-an-auth-factor/) some time ago. There are many other factors you can use for authentication, including:
+Okta [recommended moving away](https://www.okta.com/blog/2020/05/why-you-should-ditch-sms-as-an-auth-factor/) from [SMS/Voice authentication](https://www.okta.com/blog/2020/05/why-you-should-ditch-sms-as-an-auth-factor/) some time ago. There are many other factors you can use for authentication, including:
```

Comment:
```suggestion
Okta [recommended moving away from SMS/Voice authentication](https://www.okta.com/blog/2020/05/why-you-should-ditch-sms-as-an-auth-factor/) some time ago. There are many other factors you can use for authentication, including:
```

Since it goes the same link i think it should be 1 link

Reply:
It was the same link twice in a sentence in the original post as well. By fixing the link while preserving the structure of the original, I'm resisting the temptation to scope creep a broken link fix into a major overhaul of many possible improvements to the original post.